### PR TITLE
Added install-dormbase-full.sh

### PR DESCRIPTION
**Pull request by tboning**
This file should allow easy installing through virtual-env

### CODE REVIEW

File: install-dormbase-full.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+cd /dormbase
+git submodule init
+git submodule update
+# Initialize the database. This should be cleaned up.
+cd dormbase
+manage.py syncdb --migrate
+manage.py plinit
+100
+100
+yes
```
Comment (lfaraone) [inline install-dormbase-full.sh]: If this is intended to be input to `manage.py plinit`, you need to do something like this:

``` .sh
manage.py plinit << EOF
100
100
yes
[other stuff here]
EOF
cd ..
```

As is, when this script is run I get "yes\nyes\nyes\nyes" printed out.

**Comment by lfaraone** [comment]
The script does not currently function if you execute it with `bash` (or `sh` even), see the comments I made on the file itself.

Also, you should make the file executable (`chmod +x`) in this commit.

**Comment by jhurwitz** [comment]
I don't think it should be 100x100. See the sizes I gave in https://github.com/dormbase/dormbase/pull/46/files, which I got by inspecting the actual Dormbase website.

<EDIT> Oh, you do have 130x175. I didn't realize I was only looking at a subset of the diff. (I'm not used to Github!)

**Comment by tboning** [comment]
Alright, I'll fix this tomorrow. It's been a while since I touched this script and clearly it doesn't work as intended.

**Pull request merged**